bin/update-codegen.sh

```diff
@@ -7,7 +7,12 @@ set -o pipefail
 SCRIPT_DIR="$(dirname "${BASH_SOURCE[0]}")"
 SCRIPT_ROOT="$(dirname "${SCRIPT_DIR}")"
 GEN_VER=$( awk '/k8s.io\/code-generator/ { print $2 }' "${SCRIPT_ROOT}/go.mod" )
-CODEGEN_PKG=${GOPATH}/pkg/mod/k8s.io/code-generator@${GEN_VER}
+CODEGEN_PKG=target/code-generator-${GEN_VER}
+
+if [ ! -d "$CODEGEN_PKG" ]; then
```

Comment:
Petty nit: can you use double brackets, to be consistent with the other `if` below?

Reply:
yup! done.